Sources/General/main.swift

```diff
@@ -2,4 +2,67 @@
 //  Copyright © 2020 Rosberry. All rights reserved.
 //
 
-General.main()
+import GeneralKit
+import Foundation
+
+func mapCommands(runConfig: RunConfig) -> [String: CommandArguments] {
+    var result: [String: CommandArguments] = [:]
+    let arguments = Array(CommandLine.arguments.dropFirst())
+    ([runConfig.general] + runConfig.plugins).forEach { command in
+        guard let parseResult = command.parse(arguments: [command.name] + arguments),
```

Comment:
Pattern match parseResult for better naming?

Reply:
all similar things renamed to `let (matched<Type>, remainingArguments) =`